File: get-feature-40.py
import sqlite3
import json

db = sqlite3.connect('features.db')
cursor = db.cursor()
cursor.execute('SELECT id, category, name, description, steps, passes, in_progress, dependencies FROM features WHERE id = 40')
row = cursor.fetchone()
if row:
    feature = {
        'id': row[0],
        'category': row[1],
        'name': row[2],
        'description': row[3],
        'steps': json.loads(row[4]) if row[4] else [],
        'passes': bool(row[5]),
        'in_progress': bool(row[6]),
        'dependencies': json.loads(row[7]) if row[7] else []
    }
    print(json.dumps(feature, indent=2))
else:
    print("Feature #40 not found")

File: get-feature-40.php
<?php
$db = new PDO('sqlite:features.db');
$stmt = $db->prepare('SELECT id, category, name, description, steps, passes, in_progress, dependencies FROM features WHERE id = 40');
$stmt->execute();
$feature = $stmt->fetch(PDO::FETCH_ASSOC);
if ($feature) {
    $feature['steps'] = json_decode($feature['steps'], true);
    $feature['dependencies'] = json_decode($feature['dependencies'], true);
    echo json_encode($feature, JSON_PRETTY_PRINT);
} else {
    echo "Feature #40 not found\n";
}
